### PR TITLE
[DAMFreefwddyn] remove redundant forwardKin call

### PR DESCRIPTION
**Pull request by proyan**
ForwardKin is not needed after aba or computeAllTerms

### CODE REVIEW

**Comment by proyan** [comment]
Looks like I missed something.. lets wait.

**Comment by proyan** [comment]
Got it. aba doesn't do oMi updates.

File: src/multibody/actions/free-fwddyn.cpp [hunk 1]
```diff
@@ -57,6 +57,7 @@ void DifferentialActionModelFreeFwdDynamics::calc(const boost::shared_ptr<Differ
   // Computing the dynamics using ABA or manually for armature case
   if (with_armature_) {
     d->xout = pinocchio::aba(pinocchio_, d->pinocchio, q, v, d->multibody.actuation->tau);
+    pinocchio::updateGlobalPlacements(pinocchio_, d->pinocchio);
```
Comment (cmastalli) [inline src/multibody/actions/free-fwddyn.cpp]: I double check the Pinocchio code, it seems `aba` doesn't compute `oMi`. Thus, it is OK from this point of view.

But `updateGlobalPlacements` doesn't update the spatial velocities (`data.v`).
And these quantities are needed to compute the frame-velocity cost.
Therefore, the solution is to substitute it by `forwardKinematics`.

Comment (proyan) [reply src/multibody/actions/free-fwddyn.cpp]: aba updates data.v

Comment (cmastalli) [reply src/multibody/actions/free-fwddyn.cpp]: Ok, let me check in my PC.
For me, I shouldn't appear any difference in the log files.
I will come back to this in few minutes.

**Pull request merged**